frontend/src/app/core/services/user.service.ts

```diff
@@ -1,20 +1,33 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 
-import { User } from 'src/app/models/user/user';
+import { UpdateUserNamesDto } from 'src/app/models/user/update-userNames.dto';
+import { UpdateUserNotificationsDto } from 'src/app/models/user/update-userNotifications.dto';
+import { UpdateUserPasswordDto } from 'src/app/models/user/update-userPassword.dto';
+import { UserDto } from 'src/app/models/user/user-dto';
 
 import { HttpInternalService } from './http-internal.service';
 
-@Injectable({
-    providedIn: 'root',
-})
+@Injectable({ providedIn: 'root' })
 export class UserService {
-    private routePrefix = '/api/user';
+    public routePrefix = '/api/user';
```

Comment:
why not `private readonly routePrefix`?

Reply:
returned `private` (it seems changed during merges)